### ISSUE TITLE
There's an overlay at the bottom of the screen when a new message arrives

### ISSUE THREAD

**Issue by armandojrod** [moezbhatti/qksms#1074]
As seen in the screenshot, version 3.0.8

![1524275494426](https://user-images.githubusercontent.com/10835960/39079286-4ca7ad4a-44e5-11e8-8118-5cf33060a03a.jpg)

**Comment by armandojrod** [comment]
No, double checked and it's been always disabled. I use the Android default quick reply in the notification shade

**Comment by moezbhatti** [comment]
Could you attach a screenshot of this on another screen?

**Comment by armandojrod** [comment]
This one inside Flamingo

![screenshot_20180421-184511](https://user-images.githubusercontent.com/10835960/39089419-c18547a8-4594-11e8-869f-d5a82db11203.png)
